### PR TITLE
[tflitefile_tool] Use workaround for EndVector

### PR DESCRIPTION
**Pull request by hseok-oh**
Flatbuffers changes EndVector argument on 2.0.
Let's use workaround to support flatbuffers 1.12 and 2.0 for a while.

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Review by YongseopKim — DISMISSED** [review]
LGTM

File: tools/tflitefile_tool/select_operator.py [hunk 1]
```diff
@@ -22,6 +22,19 @@
 import tflite.SubGraph
 import tflite.BuiltinOptions
 import argparse
+import pkg_resources
+
+
+# On flatbuffers 2.0, EndVector doesn't requires length argument any more.
+# But flatbuffers under 2.0 (ex. 1.12) requires length argument.
+# We need this workaround until we abandon flatbuffers 1.12.
+# Refernece: https://github.com/google/flatbuffers/issues/6858
```
Comment (unterumarmung) [inline tools/tflitefile_tool/select_operator.py:31]:
```suggestion
# Reference: https://github.com/google/flatbuffers/issues/6858
```

File: tools/tflitefile_tool/select_operator.py [hunk 2]
```diff
@@ -22,6 +22,19 @@
 import tflite.SubGraph
 import tflite.BuiltinOptions
 import argparse
+import pkg_resources
+
+
+# On flatbuffers 2.0, EndVector doesn't requires length argument any more.
```
Comment (glistening) [inline tools/tflitefile_tool/select_operator.py:28]:
```suggestion
# On flatbuffers 2.0, EndVector doesn't require length argument any more.
```

**Review by glistening — DISMISSED** [review]
A tiny comment: could you please fix this one also when you fix the typo below?

**Comment by hseok-oh** [comment]
I've updated comment. PTAL

**Review by YongseopKim — APPROVED** [review]

**Review by chunseoklee — APPROVED** [review]

**Review by glistening — APPROVED** [review]

**Pull request merged**